### PR TITLE
Added Crunchy Postgres example

### LINKED ISSUE #348: Add Crunchy Postgress under Usecases
Add an example content for Crunchy Postgress

### PR DESCRIPTION
**Pull request by mahebbar**
**What this PR does / why we need it**: Add example for Crunchy Postgres

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #348

### CODE REVIEW

File: documentation/source/install/install_usecases.rst [hunk 1]
```diff
@@ -354,3 +263,40 @@ It may take some time for the pods to start as the images must be pulled and ins
...
 
 .. image:: https://raw.githubusercontent.com/openebs/openebs/master/documentation/source/_static/Jupyter.png
+
+Crunchy Postgres
+=================
+ 
+Running Crunchy Postgres on OpenEBS
+------------------------------------
+
+The following steps bring up a postgresql stateful set with one master and one replica on OpenEBS storage. OpenEBS has used centos-based postgresql containers from crunchy-data to achieve the same.
```
Comment (ksatchit) [inline documentation/source/install/install_usecases.rst]: Maybe, "This example uses centos-based postgresql containers from crunchy data to illustrate the same."

File: documentation/source/install/install_usecases.rst [hunk 2]
```diff
@@ -354,3 +263,40 @@ It may take some time for the pods to start as the images must be pulled and ins
...
+
+Download the files to your host, which has access to kubectl using the following commands.
+::
+  
+  cd $HOME
+  git clone https://github.com/openebs/openebs.git
+  cd openebs/k8s/demo/crunchy-postgres
+
+Run the StatefulSet using the following command. The files are available with default images and credentails (set.json)
+::
```
Comment (ksatchit) [inline documentation/source/install/install_usecases.rst]: The size of the openebs persistent storage volume is 400M by default. You can edit this in the storage class section of the specification file set.json 

cat set.json
..
"volumeClaimTemplates": [
      {
        "metadata": {
          "name": "pgdata"
        },
        "spec": {
          "accessModes": [
            "ReadWriteOnce"
          ],
	  "storageClassName": "openebs-basic",
          "resources": {
            "requests": {
              "storage": "400M"
            }
          }
        }
      }
..

File: documentation/source/install/install_usecases.rst (hunk 1 above)
Comment (mahebbar) [reply documentation/source/install/install_usecases.rst]: Done!

File: documentation/source/install/install_usecases.rst (hunk 2 above)
Comment (mahebbar) [reply documentation/source/install/install_usecases.rst]: Done!

**Pull request merged**